### ISSUE TITLE
[Request] Reorder home screen pages

### ISSUE THREAD

**Issue by TickingFeather** [otakuhqz/Omega#116]
## Description

[Before Android P](https://www.androidpolice.com/2018/07/18/google-removed-option-reorder-homescreens-android-ps-default-launchers/), the stock Android launcher allowed users to reorder home screen pages. 

Long-pressing the background brought up a zoomed-out view of the pages, and you could move them around just like app icons by holding+dragging (except the home page). 
![android-o-reorder-homescreen-2](https://user-images.githubusercontent.com/49458013/132134854-bc2c9b64-2c92-43a8-9cad-09380b5835ad.png)
Since the removal, custom launchers have been the solution to get this feature back. Unfortunately, Omega doesn't have it, which means you have to move several dozens of icons one by one instead.
 

## Suggested Behavior

The starting point would be to reimplement page dragging like in Android O's launcher.
Since the "long press on homescreen" is a gesture in Omega, page dragging mode could be an action in the gesture action list (just like Open widgets or Desktop bubble).

Some manufacturers like Samsung add more options to this mode, for example the ability to delete pages, or even set any page as the homepage (so you can have pages on the left too when you disable News Feed!). These can be nice to add in the future.

It would save a lot of time and hassle for users that rely on pages to organize their icons and widgets.

**Comment by ascworl** [comment]
Yes, we need this!! And 2 to 3 dock rows for storing icons. But only top row should show. Only once we swipe up from the dock should other app rows show. Like Microsoft Launcher does.

We also need backing up and restoring locally.

**Comment by otakuhqz** [comment]
I agree with both suggestions. I will work on that in the next days

**Comment by ascworl** [comment]
> I agree with both suggestions. I will work on that in the next days

Yay! Thank you! These are the last features I need so I can switch away from Microsoft Launcher (ughh). Please post here when you have included these features. I will recommend your Launcher to all my friends and family.
Where do I donate?

**Comment by Sitavi** [comment]
"the ability to delete pages, or even set any page as the homepage (so you can have pages on the left too when you disable News Feed!)"

Yes, it would be a really really great feature to have 🙏

**Comment by Merlinski999** [comment]
+1

**Comment by Arecsu** [comment]
+1 here!
